404.md

```diff
@@ -0,0 +1,24 @@
+---
+# example 404.md
+
+layout: default
+permalink: /404.html
+---
+
+# 404
+
+There is not a page at this URL.
+
+If your URL contains `/dataset` instead of `/datasets`, `/organization` instead of `/organizations`, or `/group` instead of `/categories`, you may have followed a link from an older version of Open Data Philly. In this case, we will try to redirect you, but this will not work if you have disabled JavaScript.
+
+Try searching for your dataset on the [datasets](/datasets) page.
+
+<script>
+    if (window.location.pathname.startsWith("/dataset/")) {
+        window.location.href = "/datasets" + window.location.pathname.slice(8);
+    } else if (window.location.pathname.startsWith("/organization/")) {
+        window.location.href = "/organizations" + window.location.pathname.slice(13);
+    } else if (window.location.pathname.startsWith("/group/")) {
+        window.location.href = "/categories" + window.location.pathname.slice(6);
```

Comment:
This doesn't quite seem to be working correctly; all of the categories in the old instance have the suffix `-group`, whereas the corresponding categories don't. E.g. `group/arts-culture-history-group` gets forwarded to `categories/arts-culture-history-group`, but that still returns a 404; it should be `categories/arts-culture-history`.

Reply:
thanks! I'm surprised I missed that. We should be good now!